umbraco/Umbraco.CMS.Backoffice
# Bugfix: Render collection icon for document/media tree items

## Description

This PR adds a "collection"-icon for tree items configured as a collection.

Currently, the UI only supports one icon, so if a document is protected both a collection and protected the protected state will win. (As in v13).

<img width="415" alt="Screenshot 2024-07-01 at 15 21 39" src="https://github.com/umbraco/Umbraco.CMS.Backoffice/assets/6078361/2fa11cc5-e489-43c0-8877-dbb8f5132751">

## How to test:
* Set up a document/media collection. Ensure that a "collection" icon is shown in the tree.

## Types of changes
